astropy/units/format/__init__.py

```diff
@@ -51,19 +51,14 @@ def __getattr__(name):
 
 
 def _known_formats() -> str:
```

Comment:
Now that this function is going to exported, I think we should remove the underscore - it does seem better that if in a module one defines a private variable, it is not used outside it (of course, we should not export it via `__all__`).  Same for `known_parsers`.